root/usr/local/bin/plex-sync-entrypoint

```diff
@@ -1,9 +1,11 @@
 #!/bin/sh
 
-crontab -l | grep plex-sync-job || echo 'Installing cron job' && (crontab -l 2>/dev/null; echo "$CRON_SCHEDULE /usr/local/bin/plex-sync-job >> /var/log/cron.log 2>&1") | crontab -
+export -p > /root/project_env.sh
+
+echo 'Installing cron job' && crontab -l 2>/dev/null; echo "$CRON_SCHEDULE . /root/project_env.sh >> /var/log/cron.log 2>&1; /usr/local/bin/plex-sync-job >> /var/log/cron.log 2>&1" | crontab -
```

Comment:
wouldn't this install a duplicate cron job if the container is restarted?

Reply:
From my testing no, it overwrites all previous crontab entries with the new entry

Reply:
TIL, piping to crontab from stdin replaces the contents. Sounds good then, i'll merge this in. it should publish a new image in a few minutes.

Reply:
Yea, from what I can see, it essentially inserts the new content as the entire crontab, so if you wanted to preserve it you'd want to do something like (crontab -l; echo "Cronjob line here") | crontab -
That would append a line to the current crontab rather than replacing anything in there with the one line like we do here.

Thanks for merging and publishing to dockerhub!